var/spack/repos/builtin/packages/arborx/package.py

```diff
@@ -15,7 +15,8 @@ class Arborx(CMakePackage):
 
     maintainers = ['aprokop']
 
-    version('master', branch='master')
+    version('master',   branch='master')
+    version('1.0',      commit='41dff19c319bcc23e3917fa9ed22cec3e67e5c93')
     version('0.9-beta', sha256='b349b5708d1aa00e8c20c209ac75dc2d164ff9bf1b85adb5437346d194ba6c0d')
```

Comment:
1. Not sure this should be a part of this PR
2. If it is, you would need a proper SHA hash

I'm not 100% certain whether the dependencies between `0.9-beta` and `1.0` have changed. Would prefer doing this patch later when we have properly tested all 1.0 dependencies and builds to make sure it works, as that version is pretty important.

Reply:
I can't build with 0.9-beta but only with `1.0` (and `master` for that matter). I'm happy to provide a SHA hash instead of a commit, though.